beacon-chain/sync/rpc_blob_sidecars_by_root.go

```diff
@@ -80,7 +80,7 @@ func (s *Service) blobSidecarByRootRPCHandler(ctx context.Context, msg interface
 		sc, err := s.cfg.blobStorage.Get(root, idx)
 		if err != nil {
 			if db.IsNotFound(err) {
-				log.WithError(err).Debugf("BlobSidecar not found in db, root=%x, index=%d", root, idx)
+				log.WithError(err).Debugf("Serving blob sidecar request, not found in db, root=%x, index=%d", root, idx)
```

Comment:
If the goal is to make it clear that this is untrusted input, we could make that a little more clear like "Peer requested blob sidecar by root which was not found in db".

Reply:
Done. Thanks!